staging/src/k8s.io/kubectl/pkg/cmd/testing/util.go

```diff
@@ -92,6 +92,16 @@ func TestData() (*corev1.PodList, *corev1.ServiceList, *corev1.ReplicationContro
 					EnableServiceLinks:            &enableServiceLinks,
 				},
 			},
+			{
+				ObjectMeta: metav1.ObjectMeta{Name: "foo=bar", Namespace: "test", ResourceVersion: "10"},
```

Comment:
This is actually invalid name, see https://github.com/kubernetes/kubernetes/blob/42c1ccb38ea65b27abadc9c16e1beded0ac6ac0a/pkg/apis/core/validation/validation.go#L231
Most of the resource names require it to be proper DNSName.

Reply:
Ack, I understand the name containing '=' is invalid. Thank you for clarification.